packages/calypso-products/src/plans-list.tsx

```diff
@@ -424,6 +427,15 @@ const getPlanFreeDetails = (): IncompleteWPcomPlan => ( {
 	getStoreAudience: () => i18n.translate( 'Best for students' ),
 	getPlanTagline: () =>
 		i18n.translate( 'Get a taste of the world’s most popular CMS & blogging software.' ),
+	getCustomTagline: ( flowName ) => {
```

Comment:
Do we need a new method for this? Can we use the existing `getPlanTagLine` and pass optional `flowName`?

Reply:
Thanks for addressing :-)